### ISSUE TITLE
Latest svn checkout (305) of rtmpdump is not supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#109]
```
What steps will reproduce the problem?
1. Update to latest rtmpdump svn (305)
2. Run get_flash_videos with a video site that uses rtmp

What is the expected output? What do you see instead?
Expected to download video file instead it does not.
Prints errors like:
rtmpdump: unrecognized option '--swfhash'

What version of the product are you using? On what operating system?
git 20100410
Arch Linux 2.6.33

Please provide any additional information below.
This is obviously a problem with rtmpdump updating the configuration. 
Downgrading to 2.2b for now and seems to work fine.
```

Original issue reported on code.google.com by `mjbauer95` on 13 Apr 2010 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
Seems to be a bug in the rtmpdump build system.

Try building rtmpdump with: make clean posix XCFLAGS=-DCRYPTO
```

Original comment by `monsieur...@gmail.com` on 14 Apr 2010 at 7:30
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the current rtmpdump, and rtmpdump-2.2c is now available
```

Original comment by `Highland...@gmail.com` on 16 Apr 2010 at 1:38
